src/browser/ui/ReactDOMComponent.js

```diff
@@ -232,6 +234,24 @@ function processChildContext(context, tagName) {
   return context;
 }
 
+var onlyNestTextTags = {
+  'option': true
+};
+
+function getChildrenTextContent(tag, children) {
+  var childrenContent = '';
+  ReactChildren.forEach(children, function (c) {
+    if ("production" !== process.env.NODE_ENV) {
+      ("production" !== process.env.NODE_ENV ? warning(
```

Comment:
You can just call warning directly here (the ternary operator will be added automatically as part of the build process transformations).